app/src/main/java/com/gruppo4/SMSApp/MainActivity.java

```diff
@@ -0,0 +1,140 @@
+package com.gruppo4.SMSApp;
+
+/**
+ * @author Francesco Bau'
+ */
```

Comment:
Javadoc are intended to describe classes, methods and variables, so they should be placed upon one of them.
Consider moving this block of code over your class declaration (line 18).

Reply:
Moreover, consider adding some comments on what is the purpose of this class, or, in general (considering that this is the Main Activity) what does the app.
I'm supposing that this app is used for functionalities testing and won't be deployed to users because of the heavy UI (there are 10 buttons in a single activity) and the comments in some methods.
Specifying that here will make the reviewer's life easier.

Reply:
Agreed. Thank you for pointing this out.